fe1-web/parts/lao/socialMedia/SocialFollows.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import { Text, View } from 'react-native';
+import STRINGS from 'res/strings';
+
+const SocialFollows = () => (
+  <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
```

Comment:
Wouldn't it be better to store this style in a Stylesheet ? I know this page is temporary but it is cleaner this way in my opinion.